run_dir/static/js/hashtag_csv.js

```diff
@@ -0,0 +1,567 @@
+                    this.assignments[sample_id][antibody_column] = first_sample_values[antibody_column];
+                });
+            });
+        },
+        searchProject(search_term) {
+            axios
+                .get(`/api/v1/project_search/${search_term}`)
+                .then(response => {
+                    if (response.data !== null) {
+                        this.project_suggestions = response.data;
+                    }
+                })
+                .catch(error => {
+                    this.error_messages.push('Error fetching project suggestions for search term ' + search_term + '. Please try again or contact a system administrator.');
+                    console.log(error);
+                });
+        },
+        selectProject(project_name_string) {
+            let project_id = project_name_string.split(', ')[0];
+            this.chosen_project = project_id;
+            this.search_term = '';
+            this.project_suggestions = [];
+            this.fetchProjectDetails(project_id);
+        }
+    },
+    template: 
+        /*html*/`
+        <div class="container">
+            <div class="row">
+                <div class="col-12">
+                    <h1>Hashtag CSV</h1>
+                    <p>
+                        This page enables the creation and download of a CSV file with all the hashtag information for a chosen project. 
+                        The created csv file can then be parsed by LIMS to assign sample level UDFs with corresponding antibody IDs
+                    </p>
+                    <div class="card p-3">
+                        <div class="row">
+                            <div class="col-6">
+                                <div class="form-group mb-3">
+                                    <label class="form-label" for="project">Choose project</label>
+                                    <input type="text" class="form-control" id="project" v-model="search_term" @keyup="searchProject(search_term)" placeholder="Search for project">
+                                </div>
+                                <div v-for="project in project_suggestions.slice(0, 25)" :key="project.id" class="mb-2 col-5">
+                                    <button class="btn btn-primary w-100" @click="selectProject(project.name)">
+                                        {{ project.name }}
+                                    </button>
+                                </div>
+                            </div>
+                            <div class="col-6">
+                                <div class="pl-3 py-3">
+                                    <template v-if="chosen_project">
+                                        <h1 class="mb-3">{{chosen_project}} <small>- <a class="text-decoration-none" :href="'/project/' + chosen_project">project page <i class="fa-solid fa-arrow-up-right-from-square"></i></a></small></h1>
```

Comment:
```suggestion
                                        <h1 class="mb-3">{{chosen_project}} <small>- <a class="text-decoration-none" :href="'/project/' + chosen_project" target="_blank">project page <i class="fa-solid fa-arrow-up-right-from-square"></i></a></small></h1>
```
since `fa-arrow-up-right-from-square` could be taken to mean it opens in a new page

Reply:
Accepted change.